ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,27 @@
 periodics:
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  interval: 1m
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  max_concurrency: 1
+  name: periodic-ci-image-import-to-app-ci
+  spec:
+    containers:
+    - args:
```

Comment:
This can just be 


```yaml
- args:
  - import-image
  - ci-search
  - --namespace=ci
  - --all
  command:
  - oc
```

Reply:
This is almost guaranteed to end up being extended with more imagestreams, so I think we can leave it as-is